cmd/statusd/main.go

```diff
@@ -73,6 +73,15 @@ var (
 // All general log messages in this package should be routed through this logger.
 var logger = log.New("package", "status-go/cmd/statusd")
 
+func levelFromString(level string) log.Lvl {
+	lvl, err := log.LvlFromString(strings.ToLower(level))
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Incorrect log level: %s, using defaults\n", level)
+		lvl = log.LvlInfo
```

Comment:
We can exit with an error code as well. I prefer a fail-fast approach in CLI tools.

Reply:
@adambabik yes it make sense. I've just taken this part from the old code, but if we can improve it  it's better.